src/Pirouette/Symbolic/Prover/Runner.hs

```diff
@@ -46,9 +45,9 @@ runIncorrectnessLogic ::
   Options ->
   PrtUnorderedDefs lang ->
   IncorrectnessParams lang ->
-  IncorrectnessResult lang
-runIncorrectnessLogic opts prog parms =
-  runIdentity $ execIncorrectnessLogic (proveAny opts isCounter) prog parms
+  IO (IncorrectnessResult lang)
```

Comment:
Same comment, it seems adding the `IO` wrapper here is enough for the code to compile.